app/bingey-api/server.js

```diff
@@ -10,7 +10,7 @@ const watchlistRoutes = require('$/routes/watchlist');
 const { MONGODB_HOSTNAME, MONGODB_USERNAME, MONGODB_PASSWORD } = process.env;
 
 mongoose.connect(
-  `mongodb://${MONGODB_HOSTNAME}:27017`,
+  `mongodb://${MONGODB_HOSTNAME}:27018`, // 27017 to start DB locally
```

Comment:
The remote database isn't yet setup in development, using this port breaks connecting to the database.
The comment itself is misleading, is actually needed?